pipeline/fpm/fpm.go

```diff
@@ -75,16 +75,21 @@ func create(ctx *context.Context, format, folder, arch string, binaries []contex
 	log.WithField("file", file).WithField("workdir", dir).Info("creating fpm archive")
 	var options = basicOptions(ctx, dir, format, arch, file)
 
+	bindir := "/usr/local/bin"
+	if ctx.Config.FPM.Bindir != "" {
```

Comment:
I think it would be best to set `/usr/local/bin` as a default bindir in the de defaults pipe, so we don't need the if statement here...

Reply:
Sure thing, I didn't see the defaults pipe. I'll submit a patch for that when I get home.